### DISCUSSION TITLE
Mini-slides and uncounted slides

### DISCUSSION THREAD

**Discussion by maucejo** [touying-typ/touying#76] [Q&A] [answered]
First of all, thank you for developing `touying`, which I use a lot since the "disruptive" release.

I wonder how to avoid that uncounted slides (i.e. slides defined using `freeze-slide-counter: true`) appear in the `mini-slides`.

Thank you

**Comment by OrangeX4** [comment, marked as answer]
There is no way for the time being, I will consider adding it in the next version
